README.md

```diff
@@ -10,10 +10,10 @@ PortAudio.jl is a wrapper for [libportaudio](http://www.portaudio.com/), which g
 
 ## Opening a stream
 
-The easiest way to open a source or sink is with the default `PortAudioStream()` constructor, which will open a 2-in, 2-out stream to your system's default device(s). The constructor can also take the input and output channel counts as positional arguments, or a variety of other keyword arguments.
+The easiest way to open a source or sink is with the default `PortAudioStream()` constructor, which will open a 2-in, 2-out stream to your system's default device(s). The constructor can also take the input and output channel counts as positional arguments, or a variety of other keyword arguments. If you don't specify latency and sample rate, PortAudio will use device defaults.
 
 ```julia
-PortAudioStream(inchans=2, outchans=2; eltype=Float32, samplerate=48000Hz, latency=0.1, synced=false)
+PortAudioStream(inchans=2, outchans=2; eltype=Float32, latency = nothing, samplerate = nothing)
```

Comment:
```suggestion
PortAudioStream(inchans=2, outchans=2; eltype=Float32, samplerate=48000Hz, latency=0.1, synced=false)
PortAudioStream(inchans=2, outchans=2)
```

If the goal is to show the default behavior, then I don't think we need `eltype=Float32, latency = nothing, samplerate = nothing` do we?
If the goal is to show the options, then isn't it more informative to show the options with useful values like
`samplerate=48000Hz, latency=0.1` ?
Perhaps we need two examples here, one with defaults and one with informative values?
Perhaps I am missing why you wanted to change this example?

Reply:
Oh gosh, I didn't realize that is why you were updating it.  I like your suggestion of removing Hz and synced, thereby showing the named keyword arguments.